util/global_community_block.php

```diff
@@ -17,49 +16,40 @@
  * Author: Tobias Diekershoff
  *
  * License: AGPLv3 or later, same as Friendica
- **/
-
+ */
 if ($argc != 2 || $argv[1] == "-h" || $argv[1] == "--help" || $argv[1] == "-?") {
-	echo "Usage: ".$argv[0]." [-h|profile_url]\r\n";
+	echo "Usage: " . $argv[0] . " [-h|profile_url]\r\n";
 	echo "    -h, -?, --help ... show this help\r\n";
 	echo "    profile_url ...... The URL of the profile you want to silence\r\n";
 	echo "\r\n";
 	echo "Example: block bob@example.com\r\n";
-	echo "$> ".$argv[0]." https://example.com/profiles/bob\r\n";
+	echo "$> " . $argv[0] . " https://example.com/profiles/bob\r\n";
 	echo "\r\n";
 	exit(0);
 }
 
-use Friendica\Database\DBM;
-use Friendica\Network\Probe;
+use Friendica\BaseObject;
+use Friendica\Object\Contact;
 
 require_once 'boot.php';
+
+$a = get_app();;
+BaseObject::setApp($a);
+
+require_once '.htconfig.php';
```

Comment:
This include is directly before the ```dba::connect``` so that for the shortest possible time the credentials are available in the system.

Reply:
I mirrored the order in `index.php`, I’ll revert even if I don’t fully understand the point. Credentials are available in plain text in the web root anyway?

Reply:
I would like to keep the time as short as possible in which the database credentials are available. Maybe this is some false feeling of security?

Reply:
I mean, what is the scenario where you can imagine a remote attacker would be able to obtain the variables values if they weren't unset?

Reply:
I'm a very intuitive coder. That means that I often can't explain why I'm doing things or disliking things ;-)